src/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -125,7 +125,7 @@ public async Task<AuthenticationResult> RunAsync(CancellationToken cancellationT
         {
             LogRequestStarted(AuthenticationRequestParameters);
             string accountId = AuthenticationRequestParameters.Account?.HomeAccountId?.Identifier;
-            var apiEvent = InitializeApiEvent(accountId);
```

Comment:
Apollogies for the extra noise, VS 2019 has a pretty cool code cleanup feature and one of the rules is to replace var with the actual type when that type is not easy to deduce. Pretty cool stuff.